ui/src/overridden/components/Footer/Footer.js

```diff
@@ -59,16 +59,16 @@ export const Footer = ({ ...props }) => {
               <List>
                 <List.Item>+41 22 767 24 35</List.Item>
```

Comment:
nit: should we also make the number part of config?

Reply:
The configuration is currently hardcoded in the code. The only advantage of having that config at the moment is to avoid duplications.
I would say that it make sense only if we need to re-use.